services/coverage-revision/launch.sh

```diff
@@ -3,7 +3,7 @@ set -e -x -o pipefail
 
 DST="${DST-/coverage-revision.txt}"
 
-fuzzfetch --coverage --fuzzing -a --dry-run 2>&1 | tee /dev/stderr | sed -n 's/.*> Changeset: \(.*\)/\1/p' > "$DST"
+retry fuzzfetch --coverage --fuzzing -a --dry-run 2>&1 | tee /dev/stderr | sed -n 's/.*> Changeset: \(.*\)/\1/p' > "$DST"
```

Comment:
`common.sh` isn't included in this script, so retry isn't available

Reply:
Added in [2f09b8e](https://github.com/MozillaSecurity/orion/pull/477/commits/2f09b8ed8624dd8faef42792359f6a8b6f0e3d07).